web/all_experiences.html

```diff
@@ -6,22 +6,33 @@
 <html>
 
 <head>
-    <title>Lived Religion User Experiences</title>
+    <title>All User Experiences</title>
+    <link rel="shortcut icon" type="image/x-icon" href="./images/favicon/favicon.ico">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="css/lr.css">
 </head>
 
 <body>
-    <!-- Is there a difference between the admin dashboard and the regular user dashboard -->
-    <!-- Yes.  If admin, entries can be edited -->
```

Comment:
old notes again.

Yep.  Same reason as the other hardcoded tabs.  I want there to be a placeholder that gets vaporized instead of a momentary null.  This causes the tabs to pop in and the page to bump, which was a weird UX thing.